tools/check-fenced-code.mjs

```diff
@@ -0,0 +1,54 @@
+import fs from 'fs-extra';
+import glob from 'glob';
+import MarkdownIt from 'markdown-it';
+import shell from 'shelljs';
+
+const errorTitle = 'Invalid JSON in fenced code block';
```

Comment:
Thanks that's a good idea. I'll add a summary, it might even make sense to add them individually to each error message to show up properly in the annotations.

Reply:
Cool! 😄

Reply:
I've added this now. When run in CI, the instructions are added to every annotation. When run locally, it only outputs the instructions at the end, to avoid noise.